compiler/rustc_middle/src/ty/context.rs

```diff
@@ -815,6 +824,21 @@ impl<'tcx> TyCtxt<'tcx> {
             self.def_path(def_id).to_string_no_crate_verbose()
         )
     }
+
+    #[inline]
+    pub fn par_for_each<I, T: IntoParallelIterator<Item = I> + IntoIterator<Item = I>>(
```

Comment:
`sync::par_for_each` should be used instead of adding this method.

Reply:
Done